amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/records/CreateNonElectronicRecordsTests.java

```diff
@@ -0,0 +1,133 @@
+ * (at your option) any later version.
+ * -
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * -
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.rm.community.records;
+
+import static org.alfresco.rest.rm.community.util.CommonTestUtils.generateTestPrefix;
+import static org.alfresco.utility.data.RandomData.getRandomName;
+import static org.alfresco.utility.report.log.Step.STEP;
+import static org.springframework.http.HttpStatus.CREATED;
+import static org.testng.Assert.assertNotNull;
+
+import java.util.HashMap;
+
+import org.alfresco.rest.core.v0.BaseAPI.RMProperty;
+import org.alfresco.rest.rm.community.base.BaseRMRestTest;
+import org.alfresco.rest.rm.community.model.recordcategory.RecordCategory;
+import org.alfresco.rest.rm.community.model.recordcategory.RecordCategoryChild;
+import org.alfresco.rest.v0.RMRolesAndActionsAPI;
+import org.alfresco.rest.v0.RecordsAPI;
+import org.alfresco.test.AlfrescoTest;
+import org.apache.http.HttpResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * Contains CreateNonElectronicRecords test which checks creation and basic actions(view details, edit, move, copy, delete) on non-electronic records
+ * <p/>
+ * Precondition:
+ * <p/>
+ * RM site created, contains category 1 with folder 1 and folder 2 inside it
+ * <p/>
+ * RM user has RM admin role
+ *
+ * @author Shubham Jain
+ * @Since 7.2.0 M2
+ */
+public class CreateNonElectronicRecordsTests extends BaseRMRestTest
+{
+    private RecordCategory rootCategory;
+
+    private RecordCategoryChild recordFolder;
+
+    /**
+     * data prep services
+     */
+    @Autowired
+    private RMRolesAndActionsAPI rmRolesAndActionsAPI;
+
+    @Autowired
+    private RecordsAPI recordsAPI;
+
+    private final String TEST_PREFIX = generateTestPrefix(CreateNonElectronicRecordsTests.class);
+    private final String RM_ADMIN = TEST_PREFIX + "rm_admin";
+    String category1 = TEST_PREFIX + "RM-2777 category1";
+    String folder1 = TEST_PREFIX + "RM-2777 folder1";
+    String folder2 = TEST_PREFIX + "RM-2777 folder2";
+    String recordName = "RM-2777 record";
+    String recordTitle = recordName + " title";
+    String recordDescription = recordName + " description";
+    String editedRecord = "edited RM-2777 record";
+    String editedRecordTitle = "edited RM-2777 record title";
+    String editedRecordDescription = "edited RM-2777 record description";
+
+
+    @BeforeClass (alwaysRun = true)
+    public void CreateNonElectronicRecordsTestsBeforeClass()
```

Comment:
[Really minor] I suggest renaming this to `beforeClass` or `setUp`.  If you prefer this longer name then please change the first `C` to a `c`.